readme.md

```diff
@@ -258,7 +261,7 @@ Most event listeners have a jQuery method named after them.
 
 Instead of writing `.on("click", function(){})` you can just write `.click(function(){})`;
```

Comment:
Anyway to get a short exercise here? even if it's just using `.on()` for 1 interactive event in the blog example.